app/models/manageiq/providers/vmware/infra_manager/metrics_capture.rb

```diff
@@ -390,6 +399,16 @@ def perf_query(params, counter_info, interval_name)
     return counters_by_mor, counter_values_by_mor_and_ts
   end
 
+  private
+
+  def log_targets
+    if targets.size == 1
+      "[#{target.class.name}], [#{target.id}], [#{target.name}]"
+    else
+      "[#{targets.map { |obj| obj.class.name }.uniq.join(", ")}], [#{targets.count} targets]"
```

Comment:
Minor, but prefer size in case this ever becomes an ActiveRecord relation in the future, so we don't incur an extra DB query.